protocol/0093-CFUT-product_builtin_capped_future.md

```diff
@@ -0,0 +1,63 @@
+# Built-in [Product](./0051-PROD-product.md): Capped future
+
+This built-in product provides capped futures contracts which are similar in architecture to fixed-expiry [cash settled futures](./0016-PFUT-product_builtin_future.md) with the main difference being that prices in the market are only allowed to fluctuate within the `[0, max_price]` range. This applies to all prices in the market, that's is:
+
+- order (and thus trade) prices,
+- mark prices,
+- final settlement price.
+
+The `max_price` is set at market creation time and it's immutable throughout the market's lifecycle.
+
+## 1. Product parameters
+
+Capped future should have all the product parameters and behaviour of a standard fixed-expiry cash settled [future](./0016-PFUT-product_builtin_future.md#1-product-parameters) as well as: 
+
+1. `max_price`: specifies the price cap for the market, an integer understood in the context of market decimal places,
+1. `binary_settlement`: if set to `true` settlement price other than `0` or `max_price` will be ignored,
+1. `fully_collateralised`: if set to `true` the [fully collateralised margin mode](./0019-MCAL-margin_calculator.md#fully-collateralised) is used.
```

Comment:
Ideally fully collateralised would not be a true/false flag but an alternative risk calculator / risk model that can be specified instead of log normal. (It won't need any parameters.)

**_Unless there's a technical implementation reason from @jeremyletang that we won't be doing that_** then we should describe it that way rather than as a separate parameter. This would avoid any ambiguity and any idea that a risk model could also be specified. If a log normal model is required for other aspects of the protocol it would be preferable for that to be done via separate parameters.

Reply:
Risk model is still required for price monitoring bounds and liquidity score calculation at the moment. 

How would that play out with keeping it all within standard futures product? We'd have to specify risk models separately depending on use (margin calculation, price monitoring bounds, liquidity score)?